### ISSUE TITLE
Problem with creating security domain on JBoss EAP 6.4 or Wildfly 9

### ISSUE THREAD

**Issue by Cosaquee** [coi-gov-pl/puppet-jboss#53]
When trying to run jboss 6.4  after command to setup security domain 

```
/profile=full-ha/subsystem=security/security-domain=db-auth-default/authentication=classic:add(login-modules=[{code=>"Database",flag=>"required",module-options=>[dsJndiName => "java:jboss/datasources/ds",hashStorePassword => "false",hashUserPassword => "false",principalsQuery => "select 'haslo' from uzytkownik u where u.login = upper(?)",rolesQuery => "select r.name, \'Roles\' from users u join user_roles ur on ur.user_id = u.id join roles r on r.id = ur.role_id
where u.login = ?"]}])
```

I got this error : 

```
[Error message]: {
    "outcome" => "failed",
    "rolled-back" => true,
    "result" => undefined,
    "server-groups" => undefined
}
```

This error is caused by different handling of arguments in Jboss-eap 6.4

**Comment by cardil** [comment]
Diffrences are described in JBoss EAP documentation for version 6.4 here: https://access.redhat.com/documentation/en/red-hat-jboss-enterprise-application-platform/version-6.4/how-to-configure-server-security/#using_jaas_for_securing_the_management_interfaces

**Comment by cardil** [comment]
Closed by PR merging #55
